app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
-  has_secure_password
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
```suggestion
  devise :database_authenticatable, :rememberable, :validatable
```
Giới hạn các tính năng của devise lại cho mapping với phần đang xử lý